doc/etc/reference.man

```diff
@@ -142,9 +142,3 @@ with this program. If not, see <http://www.gnu.org/licenses/>.
 .\"     -----
 .in 0
 .TC
-.\"
-.\" ----------  Include the Permuted Index  -----------------------------------
-.\"
-.bp
-.in 0
-.so etc/ref-index.so
```

Comment:
Since you made the effort to adapt the permuted index generation parts during the switch to cmake, apparently there is a use case for these entries in the PDF.

But please explain it to me, I simply don't get it :)

Reply:
To be honest, it doesn't really give much value and has become unhelpfully cluttered. At the time, I was just trying to get the documentation built as close as possible to the "current" with a goal to tidying later. So I dropped the ptx2 script. Much cleaner now.